pkg/workload/bank/bank.go

```diff
@@ -34,6 +34,7 @@ const (
 	defaultBatchSize    = 1000
 	defaultPayloadBytes = 100
 	defaultRanges       = 10
+	defaultNumTables    = 1
```

Comment:
Can you pull the bank workload changes into its own commit with a message summarizing the changes?